requirements/README.md

```diff
@@ -1,17 +1,17 @@
 
 # Requirements
 
-This is a directory of requirements files. They are maintained using `pip-tools`: 
+This is a directory of requirements files. They are maintained using `pip-tools`:
 
-* `*.in` - for developers to enter required packages 
+* `*.in` - for developers to enter required packages
 * `*.txt` - autogenerated by `pip-tools` hash-lock the specific versions for production
 
 ## Upgrading
 
-When you want to upgrade (be sure to run from main treeherder directory, this this directory)
+When you want to upgrade (be sure to run from main treeherder directory, than this directory)
 
     pip-compile --upgrade --generate-hashes --output-file requirements/common.txt requirements/common.in
     pip-compile --upgrade --generate-hashes --output-file requirements/dev.txt requirements/dev.in
```

Comment:
When adding new packages for the first time we should not be using `--upgrade`. Perhaps we need this updated on another PR.

I will redo the generating of the packages to reduce what is affected.